Mods.md

```diff
@@ -0,0 +1,16 @@
+This is a general guide about Mods for all FullScreenShenanigans projects. This guide will describe how mods and their triggers attach to these projects.
+
+### Table of Contents
+1. [ModAttachr](#modattachr)
+
+# ModAttachr
+
+ModAttachr is a module that allows for extensible triggered mod events.
+
+A mod is a custom modification that changes how the game normally runs and/or how the player interacts with the game. Each mod defines several events to run when they are fired off in the game. When an event is fired in the game, ModAttachr goes through all the enabled mods that have a defined event under that name. An event can also be triggered for only a specific mod.
```

Comment:
This was for `fireModEvent` versus `fireEvent`.
I didn't explicitly name either of them so do you want me to not mention `fireModEvent` in name/use or try describing it better?